src/js/components/LayerList.jsx

```diff
@@ -13,8 +14,8 @@ var React = require('react'),
 var MARGIN_LEFT = 10;
 
 function mapStateToProps(reduxState, ownProps) {
-  var selectedMarkId = reduxState.get('selectedMark'),
-      expandedLayers = reduxState.get('expandedLayers').toJS();
+  var selectedMarkId = getIn(reduxState, 'inspector.selected'),
+      expandedLayers = getIn(reduxState, 'inspector.expandedLayers').toJS();
```

Comment:
This utility makes handling nesting easier, @deathbearbrown it may be of use to you in the work you're doing